02_activities/assignments/assignment.sh

```diff
@@ -8,17 +8,27 @@ set -x
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
+mkdir dir1 dir2 dir3 dir4 dir5
 
 # 2. List the contents of the parent directory to verify the presence of the 5 directories
+ls
 
 # 3. Create 5 text files in dir2 named file1, file2, file3, file4, and file5
+cd dir2
+touch file1 file2 file3 file4 file5
 
 # 4. Append the words "Hello world" to dir2/file3
+echo "Hello world" > file3
 
 # 5. Verify that file3 contains the words "hello world" by printing the contents of the file in the terminal
+cat file3
 
 # 6. Delete file4
+rm file4
 
```

Comment:
missing .txt extension

should be appending text instead of overwriting the file in Q3